CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+
+PROJECT(cubrid-odbc)
+
+set (CMAKE_CXX_FLAGS "-g -Wall")
+set (CMAKE_EXE_LINK_FLAGS "-g -m64 -Wall")
+set (CMAKE_include_directories_BEFORE ON)
+
+ADD_COMPILE_OPTIONS(-g -m64 -Wall)
+
+
+INCLUDE_DIRECTORIES(cci/include)
+
+set(CUBRID_ODBC_SRCS
+	odbc_linux.c
+	unicode.c
+	odbc_catalog.c
+	odbc_connection.c
+	odbc_descriptor.c
+	odbc_diag_record.c
+	odbc_env.c
+	odbc_interface.c
+	odbc_number.c
+	odbc_result.c
+	odbc_statement.c
+	odbc_type.c
+	odbc_util.c
+)
+
+FILE(GLOB SRCS)
+
+add_definitions(-DGCC -DLINUX -D_GNU_SOURCE -DCUBRID_ODBC_UNICODE -DSQL_WCHART_CONVERT)
+
+LINK_DIRECTORIES(cci/linux_lib)
```

Comment:
I don't think you need a cci so link file in this folder.

Reply:
There are two identical so files (libcascci.so, libcascci.so.11.1). all needed?

Reply:
Actually, libcascci.so was a symbolic link to libcascci.so.11.1. It could be temporary upload of cci-lib to enable build.
I didn't decide whether linux odbc should include cci repo as a submodule or just upload compiled .so files.
This is for me to focus on the ODBC modification itself. I will rearrange it later.